tools/advisor/advisor/db_config_optimizer.py

```diff
@@ -307,6 +326,16 @@ def run(self, num_iterations):
         return new_options
 
     def run_v2(self):
+        '''
+        In every iteration of this method's optimization loop we pick ONE RULE
+        from all the triggered rules and apply all its suggestions to the
+        appropriate options. This is because this method calls
+        'apply_suggestions' which calls 'improve_db_config_v2'.
+        On the other hand, in method run(), ONE OPTION is picked in every
```

Comment:
Per our offline discussion, the comparison to to-be-deleted run() could be removed.